src/main/java/net/floodlightcontroller/devicemanager/internal/DeviceManagerImpl.java

```diff
@@ -1091,11 +1119,12 @@ protected Command processPacketInMessage(IOFSwitch sw, OFPacketIn pi, Floodlight
 
 		// Learn/lookup device information
 		Device srcDevice = learnDeviceByEntity(srcEntity);
+//		logger.info("source device is {}", srcDevice);
 		if (srcDevice == null) {
 			cntNoSource.increment();
 			return Command.STOP;
 		}
-
+//		logger.info("run here for sw {}", sw);
```

Comment:
Make sure to remove these comments. But don't be afraid to add log.debug messages if you think the information could be useful. Just be consistent

Reply:
@geddings-bsn Thanks for catch up this! This is the log help me to debug, I removed it.